### PR TITLE
Add images

### PR DESCRIPTION
**Pull request by kroenlein**
Discussing a graph model with no images makes explanations a bit unintuitive.  I added to images and a little text, though this probably needs further refinement.

### CODE REVIEW

File: docs/high-level-overview.md [hunk 1]
```diff
@@ -14,8 +14,15 @@ Conversely, there are many new ideas that exist in GEMD that are not captured by
...
 
+![Object connectivity](img/GEMD-Object-Graph.png)
+
+Don't worry if that seems quite complicated right now.
+The details are broken out in the coming sections.
 
 ## How are Data Objects defined?
+A picture is worth a thousand words:
+
+![Material Graph](img/FORMULATIONS_ILLS_2.png)
```
Comment (maxhutch) [inline docs/high-level-overview.md]: This image doesn't have any measurements in it.

File: docs/high-level-overview.md [hunk 2]
```diff
@@ -14,8 +14,15 @@ Conversely, there are many new ideas that exist in GEMD that are not captured by
 * Specified vs actual values
 * Ingredient information of input materials of a process
 
+![Object connectivity](img/GEMD-Object-Graph.png)
```
Comment (maxhutch) [inline docs/high-level-overview.md]: :+1:

File: docs/README.md [hunk 3]
```diff
@@ -1,20 +1,22 @@
 # GEMD Documentation
 
-GEMD stands for Generalized Expression of Materials Data. It's an open source format initially developed by the fine folks at Citrine Informatics. 
+GEMD stands for Generalized Expression of Materials Data. It's an open source format initially developed by the fine folks at Citrine Informatics.
+
+![Material Graph](img/FORMULATIONS_ILLS_2.png)
```
Comment (maxhutch) [inline docs/README.md]: I think this figure could do a better job of distinguishing between ingredient names and material names from the get-go, which is a consistent point of confusion.

File: docs/known-limitations.md [hunk 4]
```diff
@@ -14,8 +14,6 @@ This set of documentation describes the next generation Citrine data format, cod
 
 * The Citrine Platform does not yet support Molecular Values (InChI, SMILES) or Molecular Bounds.  
 
-* The Citrine Platform does not yet support Composition Values (Nominal Composition, Empirical Formula), although Composition Bounds are supported.
```
Comment (maxhutch) [inline docs/known-limitations.md]: :eyes:

File: docs/high-level-overview.md (hunk 1 above)
Comment (kroenlein) [reply docs/high-level-overview.md]: Supplied by Product Management.  I can look for better if this is a blocker, or we can move to better later if this is an improvement.

File: docs/README.md (hunk 3 above)
Comment (kroenlein) [reply docs/README.md]: Agree, see comment above.

File: docs/high-level-overview.md (hunk 1 above)
Comment (maxhutch) [reply docs/high-level-overview.md]: I don't think we should block on it (consistent with @djack 's point about this being strictly better) but we do need a better image.

**Pull request merged**